Modules/SubModule_ASL/xASL_wrp_ResampleASL.m

```diff
@@ -222,7 +227,18 @@ function xASL_wrp_ResampleASL(x)
     elseif round(dim4/2)~=dim4/2
         warning('Odd number of control-label pairs, skipping');
         return;
-    else    
+        
+        % Decoding of TimeEncoded data ===================
+    elseif x.modules.asl.bTimeEncoded 

```

Comment:
Is this the correct position for this function? So it occurs after motion correction, but before quantification? Should the function then be called `quant` or is it rather `im`?

Reply:
Maybe `im` is better 👍

Reply:
done